storage-proofs/porep/src/stacked/circuit/create_label.rs

```diff
@@ -177,8 +182,15 @@ mod tests {
         assert_eq!(cs.num_constraints(), 532_024);
 
         let (l1, l2) = data.split_at_mut(size * NODE_SIZE);
-        super::super::super::vanilla::create_label_exp(&graph, &id_fr.into(), &*l2, l1, node)
-            .unwrap();
+        super::super::super::vanilla::create_label_exp(
```

Comment:
Super not cool

Reply:
I believe you mean Super not chocolate ...

Reply:
Dammit, why did this change have to touch that line? @DrPeterVanNostrand I'll look into it.